.github/workflows/assign.yml

```diff
@@ -7,9 +7,11 @@ on:
 jobs:
   one:
     runs-on: ubuntu-latest
+    # toJson(github.event.comment.body) allows newline to be detected
     if: >-
       (github.event.comment.body == 'take' ||
-       github.event.comment.body == 'Take')
+       github.event.comment.body == 'take ' ||
+       toJson(github.event.comment.body) == 'take\r\n' )
```

Comment:
Maybe it would be more reliable to use a regex as in https://stackoverflow.com/a/58869470/1791279 ?  

Either way LGTM.

Reply:
I agree regex would be nicer. Although, the `if` statement was moved up to the job-level in https://github.com/scikit-learn/scikit-learn/pull/18696 , which allows for the whole workflow to be skipped without bringing up any instances.

If we start to adapt more of these "commands", I would prefer to prefix them. I've seen other project use `/` i.e. `/take`. With a prefix, it makes it more obvious that the comment was a "command".

Reply:
Makes sense.

+1 for the slash.